src/pages/Login/Login.js

```diff
@@ -0,0 +1,50 @@
+import {useState} from "react"
+import {useAuth} from "../../context"
+
+import { BiShow,BiHide } from "react-icons/bi";
+import "./login.css"
+
+export const Login = () => {
+  const [showPassword, setShowPassord] = useState(false)
+  const {loginData, setLoginData,handleLogin} = useAuth()
```

Comment:
Single you have to login from only one place instead of passing handleLogin as a function you could have passed its dispatcher